commcare_connect/organization/migrations/0007_remove_userorganizationmembership_program_manager_and_more.py

```diff
@@ -0,0 +1,21 @@
+# Generated by Django 4.2.5 on 2024-08-09 10:49
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+    dependencies = [
+        ("organization", "0006_userorganizationmembership_program_manager"),
```

Comment:
nit: it would be better to combine these migrations into one

Reply:
[](https://github.com/dimagi/commcare-connect/pull/379/commits/969a4304eeddd68eb945ebf053738ca379d52fe3) - Fixed